app/src/main/res/values/color_defs.xml

```diff
@@ -80,4 +80,5 @@
   <color name="color_def_confetti_bright_gold">#EFCF24</color>
   <color name="color_def_confetti_crystal_blue">#49A9E5</color>
   <color name="color_def_white_0">#00FFFFFF</color>
+  <color name="color_def_oppia_white_background">#F2F2F2</color>
```

Comment:
`Bbackground` word cannot be used in `color_defs` file. 
First check if similar color already exists, if not, then name it `color_def_oppia_white_2` or something of similar pattern.

Reply:
> 

Okay I'll do that.